src/quickPick.ts

```diff
@@ -136,7 +136,12 @@ interface VSCodeCommand {
 
 export type VSCodeCommandWithoutCategory = Omit<VSCodeCommand, 'category'>;
 
-async function getAllCommandPaletteCommands(): Promise<VSCodeCommandWithoutCategory[]> {
+let allCommandPaletteCommandsCached: VSCodeCommandWithoutCategory[] | undefined;
```

Comment:
`allCommandPaletteCommandsCached` seems to be duplicating `$state.allCommandPaletteCommands`.